bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitCommandExtension.java

```diff
@@ -42,6 +46,7 @@
     private static final String LEGACY_SUBCMD_PRINT_ACCESSORY = "printAccessory";
 
     private final Logger logger = LoggerFactory.getLogger(HomekitCommandExtension.class);
+    @Nullable
     private Homekit homekit;
```

Comment:
Rather than suppressing warnings everywhere I would rather you just disable null checks for this field.
```suggestion
    @NonNullByDefault({})
    private Homekit homekit;
```

Reply:
@NonNullByDefault({}) - cool trick. removing @NonNull without adding explizit @Nullable. nice. i was not aware of that one.